biicode.conf

```diff
@@ -0,0 +1,18 @@
+
+[paths]
+	# Local directories to look for headers
+	# (bii needs to look on your sources to find deps automatically)
+	src/
+
+[includes]
+	# Map any curl include to biicode's curl block
+	curl/*.h: lasote/curl/include
+
+	# Map any gtest include to biicode's gtest block
+	gtest/*.h: google/gtest/include
+	
+[requirements]
+	 google/gtest: 10
+	 lasote/curl: 2
+[parent]
+	manu343726/libappc: 0
```

Comment:
`[parent]` references the latest version available on the cloud, that is, the version your current local block comes from. When publishing, that version number (The `0` here) may change during a release, or not if you want to just fix a bug in the current version. See [`bii publish` command](http://docs.biicode.com/c++/publishing.html) in our docs.

If there's no block in the cloud available yet, there's no parent (You are working on a new block), so that number should be -1.

Ask me if you have any doubts.